legal-api/tests/unit/services/test_involuntary_dissolution.py

```diff
@@ -308,3 +308,39 @@ def test_exclude_admin_frozen_businesses(session, test_name, admin_freeze, eligi
 
     check_eligibility, eligibility_details = InvoluntaryDissolutionService.check_business_eligibility(identifier)
     assert check_eligibility == eligible
+
+
+@pytest.mark.parametrize(
```

Comment:
Given that there is only one test case, is parameterizing this test necessary? Just a thought!

Reply:
Thanks for the feedback. Mark this PR as a draft to make further updates.